packages/x-components/tailwind.config.js

```diff
@@ -3,13 +3,27 @@ const options = require('./src/tailwind/plugin-options');
 
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  content: ['./public/index.html', './src/**/*.vue'],
+  content: [
+    './public/index.html',
+    './src/**/*.vue',
+    './node_modules/@empathyco/x-tailwindcss/showcase/**/*.js'
+  ],
   prefix: 'x-',
   plugins: [xTailwindPlugin(options), oldDsPlugin],
   // In order to avoid conflicting class names with the old design system while the new XDS is being developed
-  // we are disabling all corePlugins from tailwindcss but `fontSize`, `fontWeight`, `lineHeight` and `textColor`.
+  // we are disabling all corePlugins from tailwindcss but some of them.
   // You can check that tailwindcss is still working by checking the style of the heading `Test controls`
   // below the `Start` button on the `x-components` internal demo app.
   // See https://github.com/empathyco/x/pull/655#discussion_r948923711
-  corePlugins: ['fontSize', 'fontWeight', 'lineHeight', 'textColor']
+  corePlugins: [
```

Comment:
why are we enabling more corePlugins? is it just because we missed them in the past?

Reply:
For the showcase view inside x-components. If I don't enable you will see them like this:
![image](https://user-images.githubusercontent.com/68222542/199425836-158cc1ba-cf8b-4806-a80e-404705dbe4b1.png)
instead of 
![image](https://user-images.githubusercontent.com/68222542/199425933-8738ccd2-6957-404a-8afa-24a9a01232de.png)